data/yam/autoyast/mini_desktop_remote.xml

```diff
@@ -0,0 +1,126 @@
+    <!-- minimal autoyast profile with sshd service explicitly enabled for the remote installation -->
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <email/>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+        <install_updates config:type="boolean">true</install_updates>
+        <reg_server>{{SCC_URL}}</reg_server>
+        <addons config:type="list">
+            <addon>
+                <name>sle-module-basesystem</name>
+                <version>{{VERSION}}</version>
+                <arch>{{ARCH}}</arch>
+            </addon>
+            <addon>
+                <name>sle-module-server-applications</name>
+                <version>{{VERSION}}</version>
+                <arch>{{ARCH}}</arch>
+            </addon>
+            <addon>
+                <name>sle-module-python3</name>
+                <version>{{VERSION}}</version>
+                <arch>{{ARCH}}</arch>
+            </addon>
+            <addon>
+                <name>sle-module-systems-management</name>
+                <version>{{VERSION}}</version>
+                <arch>{{ARCH}}</arch>
+            </addon>
+        </addons>
+    </suse_register>
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
+        </global>
+    </bootloader>
+    <networking>
+        <keep_install_network config:type="boolean">true</keep_install_network>
+    </networking>
+    <partitioning config:type="list">
+        <drive>
+            <device>/dev/sda</device>
+            <initialize config:type="boolean">true</initialize> 
+        </drive>
+    </partitioning>
+    <firewall config:type="map">
+        <default_zone>public</default_zone>
+        <enable_firewall config:type="boolean">true</enable_firewall>
+        <log_denied_packets>off</log_denied_packets>
+        <start_firewall config:type="boolean">true</start_firewall>
+        <zones config:type="list">
+            <zone config:type="map">
+                <description>For use in public areas. You do not trust the other computers on networks to not harm your computer. Only selected incoming connections are accepted.</description>
+                <interfaces config:type="list">
+                    <interface>eth0</interface>
+                </interfaces>
+                <masquerade config:type="boolean">false</masquerade>
+                <name>public</name>
+                <ports config:type="list"/>
+                <protocols config:type="list"/>
+                <services config:type="list">
+                    <service>dhcpv6-client</service>
+                    <service>ssh</service>
+                    <service>tigervnc</service>
+                    <service>tigervnc-https</service>
+                </services>
+                <short>Public</short>
+                <target>default</target>
+            </zone>
+        </zones>
+    </firewall>
+    <services-manager>
```

Comment:
No idea why the module named desktop and here set as multi-user.

Reply:
@jknphy do you think need to remove line 75 for this profile? or change it to graphical or keep it multi-user ?

Reply:
actually this is needed to force to boot in multi-user when we are installing a desktop, but actually we are not installing a desktop, we are missing this product module to make it similar to the other test suites in qemu, then you can keep this.